### DISCUSSION TITLE
Examples of using fife with Vue.js (frontend) FastAPI (backend)

### DISCUSSION THREAD

**Discussion by mk53202** [fief-dev/fief#245] [Q&A]
Does anyone have any examples of using Fife with a Vue.js frontend and FastAPI (backend)? I'm struggling with getting the FastAPI piece working with session/variable state (vuex) management. Any help is appreciated. Do I do the Fife client on Vue and then pass the header in through the proxied/axios calls to FastAPI with each requested? Mainly wondering if someone has some examples they can share.

**Comment by fief-bailiff** [comment]
Hail, @mk53202 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!
